core/src/main/java/kafka/autobalancer/config/AutoBalancerControllerConfig.java

```diff
@@ -68,21 +68,26 @@ public class AutoBalancerControllerConfig extends AbstractConfig {
-    public static final String AUTO_BALANCER_CONTROLLER_ACCEPTED_METRICS_DELAY_MS_DOC = "The maximum delayed time to consider a metrics valid";
-    public static final String AUTO_BALANCER_CONTROLLER_GOALS_DOC = "The goals to be detect in anomaly detector";
-    public static final String AUTO_BALANCER_CONTROLLER_ANOMALY_DETECT_INTERVAL_MS_DOC = "Time interval between anomaly detections in milliseconds";
-    public static final String AUTO_BALANCER_CONTROLLER_NETWORK_IN_USAGE_DISTRIBUTION_DETECT_THRESHOLD_DOC = "The network input bandwidth usage detect threshold in bytes";
-    public static final String AUTO_BALANCER_CONTROLLER_NETWORK_IN_DISTRIBUTION_DETECT_AVG_DEVIATION_DOC = "The acceptable range of deviation for average network input bandwidth usage";
-    public static final String AUTO_BALANCER_CONTROLLER_NETWORK_OUT_USAGE_DISTRIBUTION_DETECT_THRESHOLD_DOC = "The network output bandwidth usage detect threshold in bytes";
+    public static final String AUTO_BALANCER_CONTROLLER_ACCEPTED_METRICS_DELAY_MS_DOC = "The Controller's tolerable Broker metrics reporting delay, in milliseconds, has a default value of 60 seconds. " +
+        "When the Controller checks for data rebalancing, if the most recent metrics reporting interval of the Broker exceeds the configured value, the Controller considers the state of that Broker to be based on outdated data, " +
+        "and does not perform data rebalancing on that Broker. This configuration cannot be less than the Broker metrics reporting interval.";
+    public static final String AUTO_BALANCER_CONTROLLER_GOALS_DOC = "The goal list is to be detected in the anomaly detector.";
+    public static final String AUTO_BALANCER_CONTROLLER_ANOMALY_DETECT_INTERVAL_MS_DOC = "The Controller checks whether data rebalancing is necessary at a minimum interval, in milliseconds, with a default value of 60 seconds. " +
+        "The actual time to check for the next data rebalancing is also related to the scale of the partition migration that occurred this time. The more partitions that are migrated, the longer the interval for the next data rebalancing check." +
+        "Shortening the minimum check interval can improve the sensitivity of data rebalancing, but in actual use, it should be configured in conjunction with the Broker metrics reporting interval to avoid the Controller being unable to perceive the result of the most recent scheduling, thereby causing duplicate scheduling.";
+    public static final String AUTO_BALANCER_CONTROLLER_NETWORK_IN_USAGE_DISTRIBUTION_DETECT_THRESHOLD_DOC = "Network in monitoring threshold. If it is below this threshold, the load of this Broker will not be actively optimized.";
+    public static final String AUTO_BALANCER_CONTROLLER_NETWORK_IN_DISTRIBUTION_DETECT_AVG_DEVIATION_DOC = "The NetworkInUsageDistributionGoal tries to adjust the network inbound bandwidth utilization rate of each broker to a traffic distribution range." +
```

Comment:
The acceptable range of deviation for average network input bandwidth usage. Default is 0.2, which means the expected network traffic range will be [0.8 * loadAvg, 1.2 * loadAvg]